NEWS.md

```diff
@@ -33,6 +33,117 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## [unreleased]
+
+The eye-catching new features of this release are undoubtedly:
+1. Support for the `ext-workspace` protocol with big thanks to @Consolatis
+2. Menu enhancements including icons and dynamic root-menus. Credits here go to
+   @Consolatis, @tokyo4j and @johanmalm for improvements to both backend buffer
+   managements and front end menu mechanics.
+
+However - on the whole - the main effort of this release has gone into
+stability, usability and performance fixes, and it really feels like we have
+matured nicely against the wlroots 0.18 series.
+
+Notes to package maintainers:
+- This version introduces the ext-workspace protocol which breaks xfce4-panel
+  4.20.0. Subsequent releases has a fix for this so make sure xfce4-panel is
+  shipped at >= 4.20.1.
+
+### Added
+
+- Add config options `<resize><cornerRange>` and `<resize><minimumArea>` [#2529]
+- Menu icons support [#2509]
+
+```xml
+<menu>
+  <showIons>yes</showIcons>
+</menu>
+```
+
+- Support toplevel pipemenus [#2238] [#2239]
+
+```xml
+<?xml version="1.0"?>
+<openbox_menu>
+  <menu id="root-menu" label="" execute="labwc-menu-generator -p -I" />
+</openbox_menu>
+```
+
+- Add `<theme><fallbackAppIcon>` to specify the icon name to be used when
+  lookups for an application icon is failed [#2518]
```

Comment:
"is failing" or "has failed"?
"is failed" sounds odd to me

Reply:
s/is/has/ IMHO

Reply:
Thanks. Went for "have"